docs/PayPal/integration.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Pay with PayPal Custom Integration
+keywords: 
+contentType: docs
+productStatus: current
+apiVersion: TODO
+sdkVersion: TODO
+---
+
+# Pay with PayPal Custom Integration
+
+Follow these steps to add Card payments.
+
+1. [Know before you code](#know-before-you-code)
+1. [Add PayPal Payments](#add-paypal-payments)
+1. [Test and go live](#test-and-go-live)
+
+## Know before you code
+
+You will need to set up authorization to use the PayPal Payments SDK. 
+Follow the steps in [Get Started](https://developer.paypal.com/api/rest/#link-getstarted) to create a client ID. 
+
+You will need a server integration to create an order and capture the funds using [PayPal Orders v2 API](https://developer.paypal.com/docs/api/orders/v2). 
+For initial setup, the `curl` commands below can be used in place of a server SDK.
```

Comment:
Just to clarify, all of the curl commands used below will allow you to get setup without doing the two prerequisites above?

Reply:
No - the curl commands allow you not to create a server integration, but you still need to do the step on line 21. Could I phrase that somehow that makes it more clear?

Reply:
🤔 How about something like this: 
```
Required:
You will need to set up authorization to use the PayPal Payments SDK. 
Follow the steps in [Get Started](https://developer.paypal.com/api/rest/#link-getstarted) to create a client ID. 

Optional: 
- When creating your own server integration use [PayPal Orders v2 API](https://developer.paypal.com/docs/api/orders/v2). However, if you would like to forego this, you can use the provided `curl` commands given with each relevant step.
```

Not sure on the optional/required aspect, but wondering how to best separate them.

Reply:
Hmm.. Not sure about that because it isn't really optional. You have to use something to create the order (for now it's curl commands but hopefully when server SDKs exist that will be an option)

Reply:
Ah fair enough. Maybe...
```
In lieu of a server integration that uses [PayPal Orders v2 API](https://developer.paypal.com/docs/api/orders/v2), you can use the provided curl commands to validate your integration.
```
Something like that? Then nix the optional/required bits?